recipes/natural_language_processing/codegen/README.md

```diff
@@ -22,111 +31,131 @@ In order to build this application we will need a model, a Model Service and an
 * [Interact with the AI Application](#interact-with-the-ai-application)
 * [Embed the AI Application in a Bootable Container Image](#embed-the-ai-application-in-a-bootable-container-image)
 
-### Download a model
+
+## Quickstart
+To run the application with pre-built images from `quay.io/ai-lab`, use `make quadlet`. This command
+builds the application's metadata and generates Kubernetes YAML at `./build/codegen.yaml` to spin up a Pod that can then be launched locally.
+Try it with:
+
+```
+make quadlet
+podman kube play build/codegen.yaml
+```
+
+This will take a few minutes if the model and model-server container images need to be downloaded. 
+The Pod is named `codegen`, so you may use [Podman](https://podman.io) to manage the Pod and its containers:
+
+```
+podman pod list
+podman ps
+```
+
+Once the Pod and its containers are running, the application can be accessed at `http://localhost:8501`. 
+Please refer to the section below for more details about [interacting with the codegen application](#interact-with-the-ai-application).
+
+To stop and remove the Pod, run:
+
+```
+podman pod stop codegen
+podman pod rm codgen
+```
+
+## Download a model
 
 If you are just getting started, we recommend using [Mistral-7B-Instruct](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.1). This is a well
 performant mid-sized model with an apache-2.0 license. In order to use it with our Model Service we need it converted
 and quantized into the [GGUF format](https://github.com/ggerganov/ggml/blob/master/docs/gguf.md). There are a number of
 ways to get a GGUF version of Mistral-7B, but the simplest is to download a pre-converted one from
 [huggingface.co](https://huggingface.co) here: https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF.
 
-There are a number of options for quantization level, but we recommend `Q4_K_M`.
+There are a number of options for quantization level, but we recommend `Q4_K_M`. 
 
 The recommended model can be downloaded using the code snippet below:
 
 ```bash
-cd models
-wget https://huggingface.co/TheBloke/Mistral-7B-Code-16K-qlora-GGUF/resolve/main/mistral-7b-code-16k-qlora.Q4_K_M.gguf
-cd ../
+cd ../../../models
+curl -sLO https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF/resolve/main/mistral-7b-instruct-v0.1.Q4_K_M.gguf
```

Comment:
Is this intentional changing the model name?

Reply:
nope, good catch.